docs/articles/documentation/test-api/identify-the-browser-and-platform.md

```diff
@@ -0,0 +1,295 @@
+---
+layout: docs
+title: Identify the Browser and Platform
+permalink: /documentation/test-api/identify-the-browser-and-platform.html
+checked: true
+---
+# Identify the Browser and Platform
+
+TestCafe allows you to obtain information about the current user agent in test code. These data identify the operating system, platform type, browser, engine, etc.
+
+Use the `t.browser` property to access user agent data.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `My fixture`
+    .page `https://example.com`;
+
+test('My test', async t => {
+    if (t.browser.name !== 'Chrome')
+        await t.expect(Selector('div').withText('Browser not supported').visible).ok();
+});
+```
+
+`t.browser` exposes the following properties:
+
+Property | Description   | Example
+-------- | ------------- | -------
+[alias](#alias) | The browser alias string specified when tests were launched. | `firefox:headless`
+[name](#name) | The browser name. | `Chrome`
+[version](#version) | The browser version. | `77.0.3865.120`
+[platform](#platform) | The platform type. | `desktop`
+[headless](#headless) | `true` if the browser runs in headless mode.
```

Comment:
Should we add an example value to this row?
```suggestion
[headless](#headless) | `true` if the browser runs in headless mode. | `true`
```

Reply:
Oh, I just forgot the `Type` column. As long as we have `Boolean` there, the example is not really needed. If only for consistency.